import_busstops.py

```diff
@@ -7,6 +7,7 @@ class BusImportCommand(CSVImportCommand):
 
     def __init__(self, *kargs, **kwargs):
         self.dry_run = kwargs.pop('dry_run')
+        kwargs['num_expected_records'] = 430525
```

Comment:
All right, no problem. If you can just fix merge conflicts I can review it again and merge, cheers